tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -59,7 +60,7 @@ def _validate_type(self):
 
     return True
 
-  def IsValidEntityInstance(self):
+  def is_valid_entity_instance(self):
```

Comment:
can you please keep this as before? with caps. 
Example is RecusriveDirWalk

Reply:
`gpylint` complains that it doesn't follow the style of the other functions; should I change the other functions in this file to be CamelCase to see if that works?

Reply:
check the https://github.com/google/digitalbuildings/blob/master/tools/rdf_generator/rdfformat/generator/yaml_handler.py

Reply:
Done